plugins/package-managers/node/src/funTest/assets/projects/synthetic/npm/babel-expected-output.yml

```diff
@@ -15,7 +15,7 @@ project:
     type: "Git"
     url: "<REPLACE_URL_PROCESSED>"
     revision: "<REPLACE_REVISION>"
-    path: "plugins/package-managers/node/src/funTest/assets/projects/synthetic/npm/babel"
+    path: "<REPLACE_PATH>"
```

Comment:
Commit message: Please explain why. As the test seems to have worked before, what's the improvement now?

Reply:
I'm still not convinced. `<REPLACE_...>` patterns in general should *only* be used for stuff that cannot be hard-coded in expected results, that's why the replace mechanism was introduced.

So if we're using `<REPLACE_PATH>` (or any other `<REPLACE_...>` pattern) where hard-coding would be possible in the expected result, we should hard-code instead.

Reply:
This would be a change in the approach, just consider the `REPLACE_DEFINITION_FILE_PATH`. That's probably not necessary in any case but we have it. This commit is only about making things (locally) consistent. I'm not against changing the approach per se, but not in this PR / commit.

BTW: The use of these patterns makes it easier to rename files, or to move them.

Reply:
> The use of these patterns makes it easier to rename files, or to move them.

So *that* is the argument that convinces me, and that I would have liked to read in the commit message. I'll approve.